prow/config/config.yaml

```diff
@@ -1369,6 +1369,18 @@ branch-protection:
                   - 'idc-jenkins-ci-tiflash/build'
                   - 'license/cla'
                 strict: true
+            planner_refactory:
+              protect: true
+              required_status_checks:
+                contexts:
+                  - 'tide'
+                  - 'idc-jenkins-ci-tiflash/unit-test'
+                  - 'idc-jenkins-ci-tiflash/integration-test'
+                  - 'idc-jenkins-ci-tiflash/build'
+                  - "check-issue-triage-complete"
```

Comment:
remove it. the issue-triage plugin only works on `master`, I think the bug issues should not be fixed on the feature branch, so it doesn't need to triage.